app/config.py

```diff
@@ -81,9 +81,6 @@ class Config(object):
     # Firetext simluation key
     LOADTESTING_API_KEY = os.getenv("LOADTESTING_API_KEY")
 
-    # Hosted graphite statsd prefix
-    STATSD_PREFIX = os.getenv('STATSD_PREFIX')
```

Comment:
i think we still need this for apps we haven't switched over to prometheus? We're still including it in the manifest and it looks like it's still being used in https://github.com/alphagov/notifications-utils/blob/master/notifications_utils/clients/statsd/statsd_client.py#L20

Reply:
Oops, yeah, it absolutely should still be there, good catch!